v2/internal/frontend/assetserver/assethandler.go

```diff
@@ -150,6 +151,8 @@ func (d *assetHandler) serveFSFile(rw http.ResponseWriter, req *http.Request, fi
 		if err != nil {
 			return err
 		}
+	} else if isDirectoryPath {
+		return fmt.Errorf("a file has been requested with a trailing slash, please remove the trailing slash from your request")
```

Comment:
Same as above

Reply:
This would result in the same problem but in reverse order, a file in the root folder would be returned as file in a sub folder. So relative references would then be loaded from the sub folder instead of the root.